docs/styling/theming.md

```diff
@@ -5,7 +5,7 @@ from pcweb.pages.docs import library
 
 # Theming
 
-As of Reflex `v0.4.0`, you can now theme your Reflex applications. The core of our theming system is directly based on the [Radix Themes](https://radix-ui.com/docs/themes) library. This allows you to easily change the theme of your application along with providing a default light and dark theme.
+As of Reflex `v0.4.0`, you can now theme your Reflex applications. The core of our theming system is directly based on the [Radix Themes](https://www.radix-ui.com) library. This allows you to easily change the theme of your application along with providing a default light and dark theme.
```

Comment:
Maybe specify in the last sentence what theme actually means, all the components will have a unified color appearance